reference/5.1/PowershellGet/Uninstall-Script.md

```diff
@@ -12,36 +12,52 @@ title: Uninstall-Script
 # Uninstall-Script
 
 ## SYNOPSIS
-Uninstalls a script file.
+Uninstalls a script.
 
 ## SYNTAX
 
 ### NameParameterSet (Default)
+
 ```
 Uninstall-Script [-Name] <String[]> [-MinimumVersion <String>] [-RequiredVersion <String>]
  [-MaximumVersion <String>] [-Force] [-AllowPrerelease] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ### InputObject
+
 ```
 Uninstall-Script [-InputObject] <PSObject[]> [-Force] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Uninstall-Script** cmdlet uninstalls the specified script files from the online gallery.
+
+The `Uninstall-Script` cmdlet uninstalls a specified script from the local computer.
 
 ## EXAMPLES
 
 ### Example 1: Uninstall a script file
+
+This example uninstalls a script from the local computer.
+
+```powershell
+Get-InstalledScript -Name UpdateManagement-Template
```

Comment:
We don't need to call Get-InstalledScript if we are not using the pipeline. Think we should have a simple example to show uninstall by name and another to show using the pipeline.

Reply:
Thanks for the feedback. I updated the examples.